resources/analysisTools/qcPipeline/bwaErrorChecking.sh

```diff
@@ -11,55 +11,56 @@
 useMBufferStreaming="${useMBufferStreaming-false}"
 
 # Test for success before renaming!
-success=$(grep " fault" "$FILENAME_BWA_LOG")
-[[ ! -z "$success" ]] && throw 31 "found segfault '$success' in bwa logfile!"
+failure=$(grepIgnoreEmpty " fault" "$FILENAME_BWA_LOG")
```

Comment:
These could all use the same base function.

Reply:
Also because of the grep.
could be something like:
```
failOnBWAError($FILENAME, $needle, $failmessage)
```
This could improve readability.